libraries/Microsoft.Bot.Builder.Ai/QnAMaker.cs

```diff
@@ -15,13 +15,14 @@ public class QnAMaker
         public const string APIManagementHeader = "Ocp-Apim-Subscription-Key";
         public const string JsonMimeType = "application/json";
 
+        private static HttpClient g_httpClient = new HttpClient();
```

Comment:
why not create an instance level HttpClient when not specified rather than a global static one that is to be shared across instances of your QnAMaker claass?